x-pack/legacy/plugins/ml/public/overview/components/anomaly_detection_panel/table.tsx

```diff
@@ -0,0 +1,193 @@
+}
+
+interface Props {
+  items: Groups;
+  statsBarData: JobStatsBarStats;
+  jobsList: Jobs;
+}
+
+export const AnomalyDetectionTable: FC<Props> = ({ items, jobsList, statsBarData }) => {
+  const groupsList = Object.values(items);
+  const [pageIndex, setPageIndex] = useState(0);
+  const [pageSize, setPageSize] = useState(10);
+
+  const [sortField, setSortField] = useState<string>(AnomalyDetectionListColumns.id);
+  const [sortDirection, setSortDirection] = useState<SortDirection>(SORT_DIRECTION.ASC);
+
+  // columns: group, max anomaly, jobs in group, latest timestamp, docs processed, action to explorer
+  const columns: any[] = [
+    {
+      field: AnomalyDetectionListColumns.id,
+      name: i18n.translate('xpack.ml.overview.anomalyDetection.tableId', {
+        defaultMessage: 'Group ID',
+      }),
+      render: (id: Group['id']) => <JobSelectorBadge id={id} isGroup={id !== 'ungrouped'} />,
+      sortable: true,
+      truncateText: true,
+      width: '20%',
+    },
+    {
+      field: AnomalyDetectionListColumns.maxAnomalyScore,
+      name: i18n.translate('xpack.ml.overview.anomalyDetection.tableMaxScore', {
+        defaultMessage: 'Max anomaly score',
+      }),
+      sortable: true,
+      render: (score: Group['max_anomaly_score']) => {
+        if (score === null) {
+          return <EuiLoadingSpinner />;
+        } else {
+          const color: string = getSeverityColor(score);
+          return (
+            // @ts-ignore
+            <EuiHealth color={color} compressed="true">
+              {score >= 1 ? Math.floor(score) : '< 1'}
+            </EuiHealth>
+          );
+        }
+      },
+      truncateText: true,
+      width: '150px',
+    },
+    {
+      field: AnomalyDetectionListColumns.jobIds,
+      name: i18n.translate('xpack.ml.overview.anomalyDetection.tableNumJobs', {
+        defaultMessage: 'Jobs in group',
+      }),
+      render: (jobIds: Group['jobIds']) => jobIds.length,
+      sortable: true,
+      truncateText: true,
+      width: '100px',
+    },
+    {
+      field: AnomalyDetectionListColumns.latestTimestamp,
+      name: i18n.translate('xpack.ml.overview.anomalyDetection.tableLatestTimestamp', {
+        defaultMessage: 'Latest timestamp',
+      }),
+      dataType: 'date',
+      render: (time: number) => formatHumanReadableDateTimeSeconds(time),
+      textOnly: true,
+      sortable: true,
+      width: '20%',
+    },
+    {
+      field: AnomalyDetectionListColumns.docsProcessed,
+      name: i18n.translate('xpack.ml.overview.anomalyDetection.tableDocsProcessed', {
+        defaultMessage: 'Docs processed',
+      }),
+      render: (docs: number) => toLocaleString(docs),
+      textOnly: true,
+      sortable: true,
+      width: '20%',
+    },
+    {
+      name: i18n.translate('xpack.ml.overview.anomalyDetection.tableActionLabel', {
+        defaultMessage: 'Actions',
+      }),
+      render: (group: Group) => <ExplorerLink jobsList={getJobsFromGroup(group, jobsList)} />,
```

Comment:
Seen this strange behavior where the count in the table is different to the count in the tooltip on the link to the Anomaly Explorer (16 is the correct number):

![image](https://user-images.githubusercontent.com/7405507/65152972-db9fc380-da20-11e9-8e63-655bd08b7385.png)

Reply:
Fixed with 59c980893ca72c667bda1921584b74d5907f35bf